src/parse.py

```diff
@@ -30,13 +30,16 @@ def open_json_file(filename):
         raise ValueError('{} is not valid JSON.'.format(filename))
 
 
-def validate_object_keys(keys, json):
+def validate_object_keys(keys, test_json):
     try:
         assert isinstance(keys, (list, tuple))
+        found_keys_list = []
         for key in keys:
-            if key in json.keys():
+            if key in test_json.keys():
+                found_keys_list.append(key)
                 continue
             raise Exception('Key is missing')
+        return found_keys_list
```

Comment:
there is no need to return the dictionary as it will always return the same list or fail. Just true is enough.